src/components/Rocket.js

```diff
@@ -18,15 +19,22 @@ const Rocket = (props) => {
     dispatch(reserveRocket(id));
   };
 
+  const cancelRocketHandler = () => {
+    dispatch(cancelRocket(id));
+  };
+
   return (
     <Row id={id} className="my-4">
       <Col md={5}>
         <img src={img} alt="rocket img" className="w-100" />
       </Col>
       <Col>
         <p className="rocket-name">{names}</p>
-        <p className="rocket-desc">{description}</p>
-        <Button onClick={reserveRocketHandler} variant={reserved ? 'outline-secondary' : 'primary'}>{reserved ? 'Cancel Reservation' : 'Reserve Rocket' }</Button>
+        <p className="rocket-desc">
+          <span className={(reserved) ? 'reserve-span' : ' '}>{(reserved) ? 'Reserved' : null }</span>
+          {description}
+        </p>
+        <Button onClick={(reserved) ? cancelRocketHandler : reserveRocketHandler} variant={reserved ? 'outline-secondary' : 'primary'}>{reserved ? 'Cancel Reservation' : 'Reserve Rocket' }</Button>
```

Comment:
Do you think you have to add a different handler for canceling? because it can be done with just one event handler and you can switch the reserved state with that.